Configuration/TCA/tx_variables_marker.php

```diff
@@ -136,6 +143,7 @@
                 'rows' => 4,
                 'eval' => 'required,trim'
             ],
+            'defaultExtras' => $settings['enableRte']?'richtext:rte_transform[mode=ts_css]':'',
```

Comment:
```
$settings['enableRte']
```
May not be an existing index. I know TYPO3 will not complain but nevertheless we should avoid that IMO:

+ PSR-2 formatting needed:

```
$enableRte ? 'richtext:rte_transform[mode=ts_css]' : '',
```